src/Caching/Hybrid/src/Runtime/IBufferDistributedCache.cs

```diff
@@ -0,0 +1,50 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Buffers;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Microsoft.Extensions.Caching.Distributed; // intentional for parity with IDistributedCache
+
+/// <summary>
+/// Represents a distributed cache of serialized values, with support for low allocation data transfer.
+/// </summary>
+public interface IBufferDistributedCache : IDistributedCache
+{
+    /// <summary>
+    /// Attempt to retrieve an existing cache item.
+    /// </summary>
+    /// <param name="key">The unique key for the cache item.</param>
+    /// <param name="destination">Target to write the cache contents on success.</param>
+    /// <returns><c>True</c> if the cache item is found, <c>False</c> otherwise.</returns>
+    /// <remarks>This is functionally similar to <see cref="IDistributedCache.Get(string)"/>, but avoiding the array allocation.</remarks>
```

Comment:
```suggestion
    /// <param name="destination">The target to write the cache contents on success.</param>
    /// <returns><c>true</c> if the cache item is found, <c>false</c> otherwise.</returns>
    /// <remarks>This is functionally similar to <see cref="IDistributedCache.Get(string)"/>, but avoids the array allocation.</remarks>
```